src/runtime/hexagon/ops/conv2d.h

```diff
@@ -144,7 +207,42 @@ void blockize_hwc_16b(void* out, void* inp_flat, int height, int width, int dept
  * @param width
  * @param depth
  */
-void deblockize_hwc_16b(void* out_flat, void* inp, int height, int width, int depth);
+template <typename T, int block_height, int block_width, int block_depth>
+void deblockize_hwc(void* out_flat, void* inp, int height, int width, int depth) {
```

Comment:
Would it make sense for the type of `inp` to be `const void*`?

Reply:
I'll add the `const void*`, it makes sense, thanks.